python/az/aro/azext_aro/custom.py

```diff
@@ -279,6 +289,13 @@ def get_network_resources(cli_ctx, subnets, vnet):
     return resources
 
 
+def get_disk_encryption_resources(oc):
+    disk_encryption_set = oc.master_profile.disk_encryption_set_id
+    resources = set()
```

Comment:
After a discussion, the set() is used to keep code consistency even when there will be one object. This makes code more streamlined.